File: All_File/c++/prime.cpp
#include<iostream>
using namespace std;
int main()
{
	int n,i,m=0,flag=0;
	cout<<"Enter the number to chake prime:";
	cin>>n;
	m=n/2;
	for(i=2;i<=m;i++)
	{
		if(n%i==0)
		{
			cout<<"Number is not prime"<<endl;
			flag=1;
			break;
		}
	}
	if(flag==0)
	cout<<"Number is prime"<<endl;
	return 0;
}

File: All_File/Delotiee2/Prime.java
//3  5  7  11 13 17 19 23

public class Prime {
    public static void main(String[] args) {
        int n = 21,count=0;
        for (int i = 2; i < n / 2; i++) {
            if (n % i == 0) {
                count++;
                break;  
            }
        }
        if (count == 1) {
            System.out.println("not prime");
        } else {
            System.out.println("prime Number");
        }
    }
}
